hydra-access-controls/app/models/hydra/permissions_solr_document.rb

```diff
@@ -3,8 +3,8 @@ def under_embargo?
     #permissions = permissions_doc(params[:id])
     embargo_key = Hydra.config.permissions.embargo.release_date
     if self[embargo_key]
-      embargo_date = Date.parse(self[embargo_key].split(/T/)[0])
-      return embargo_date > Date.parse(Time.now.to_s)
+      embargo_date = DateTime.parse(self[embargo_key])
+      return embargo_date > DateTime.parse(Time.now.to_s)
```

Comment:
Yes @atz the revision looks okay to me. Sorry didn't see the ping message or the comments earlier.